code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+                DeviceWithHardwareAddressAndNetworks:
+                  $ref: "#/components/examples/DeviceWithHardwareAddressAndNetworks"
+                DeviceWithServiceSiteAndNetworks:
+                  $ref: "#/components/examples/DeviceWithServiceSiteAndNetworks"
+                DeviceWithHardwareAddressAndServiceSiteAndNetworks:
+                  $ref: "#/components/examples/DeviceWithHardwareAddressAndServiceSiteAndNetworks"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/DeviceNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /devices/{deviceId}/isolated-networks:
+    parameters:
+      - $ref: "#/components/parameters/deviceId"
+    get:
+      tags:
+        - Devices
+        - Isolated Networks
+      summary: Get all networks configured on a device
+      operationId: getNetworksByDevice
+      description: |
+        Returns all networks configured on a device that matches the given ID.  Scopes impact the output of this endpoint.  See the examples for various permutations.
+
+        **Required Scopes:**
+          - **any of**
+            - `cpe-management-devices-read`
+            - `cpe-management-devices-write`
+            - `cpe-management-devices-hardware-address-read`
+          - **any of**
+            - `cpe-management-isolated-networks-read`
+            - `cpe-management-isolated-networks-write`
+            - `cpe-management-isolated-networks-primary-read`
+            - `cpe-management-isolated-networks-primary-write`
+            - `cpe-management-isolated-networks-auxiliary-read`
+            - `cpe-management-isolated-networks-auxiliary-write`
+      responses:
+        "200":
+          description: Contains information about networks configured on the device
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NetworkList"
+              examples:
+                NetworkListWithDevices:
+                  $ref: "#/components/examples/NetworkListWithDevices"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/DeviceNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+    patch:
+      tags:
+        - Devices
+        - Isolated Networks

```

Comment:
```suggestion
```

Reply:
Agree with keeping this one devices since it is a device operation.